packages/manager/apps/communication/src/data/api/contacts.ts

```diff
@@ -0,0 +1,61 @@
+import apiClient from '@ovh-ux/manager-core-api';
+import {
+  ContactMean,
+  CreateContactMean,
+  UpdateContactMean,
+  ValidateContactMean,
+} from '../types/contact-mean.type';
+
+export const createContactMean = async (data: CreateContactMean) => {
+  const { data: responseData } = await apiClient.v2.post<ContactMean>(
+    '/notification/contactMean',
+    data,
+  );
+  return responseData;
+};
+
+export const deleteContactMean = async (contactMeanId: string) => {
+  const { data: responseData } = await apiClient.v2.delete<ContactMean>(
+    `/notification/contactMean/${contactMeanId}`,
+  );
+  return responseData;
+};
+
+export const updateContactMean = async ({
+  contactMeanId,
+  data,
+}: {
+  contactMeanId: string;
+  data: UpdateContactMean;
+}) => {
+  const { data: responseData } = await apiClient.v2.put<ContactMean>(
+    `/notification/contactMean/${contactMeanId}`,
+    data,
+  );
+  return responseData;
+};
+
+export const validateContactMean = async (
+  contactMeanId: string,
+  data: ValidateContactMean,
+) => {
+  const { data: responseData } = await apiClient.v2.post<ContactMean>(
+    `/notification/contactMean/${contactMeanId}/validate`,
+    data,
+  );
+  return responseData;
+};
+
+export const restartValidationContactMean = async (contactMeanId: string) => {
+  const { data: responseData } = await apiClient.v2.post<ContactMean>(
+    `/notification/contactMean/${contactMeanId}/restartValidation`,
+  );
+  return responseData;
+};
+
+export const getContactMean = async (contactMeanId: string) => {
```

Comment:
Can we create query key for each calls here ? It will be easier to re use and will help better maintainability if we are going to update it. 
```suggestion

export const getContactMeanQueryKey = (params: contactMeanId) => [
  `/notification/contactMean/${contactMeanId}`,
];

export const getContactMean = async (contactMeanId: string) => {
```